components/build-service/staging/base/manager_resources_patch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: build-service-controller-manager
+  namespace: build-service
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        resources:
+          limits:
+            cpu: 500m
+            memory: 1Gi
+          requests:
+            cpu: 150m
+            memory: 512Mi
```

Comment:
We are running performance test on staging, we should have identical values between staging and prod. Please set same values for mem/cpu requests/limits as for prod

Reply:
ok, will do